app/views/content/landing/how-much-do-teachers-get-paid.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "How much do teachers get paid?"
+description: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
@philspray @EllieNodder Need a meta description (even though the page is noindexed)?

Reply:
hmm do we? I wouldn't have said so - feel free to disagree @philspray!

Reply:
I guess no one is going to see it, but I would prefer to have something relevant in there rather than Lorem ipsum. :)

Reply:
Agree - think it might be worth having something if quick to write in case a) noindexing is ignored (hope not!) but also if this is used to pull through a description when linking to the page in future (either internally or via social)

Reply:
we could just remove the description? but up to you!

Reply:
I've added some words :)